### PR TITLE
Added Indonesian translation

### PR DESCRIPTION
**Pull request by rilut**
Hi,
I added the Indonesian translation.

Thanks.

### CODE REVIEW

File: config/locales/id.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+en:
```
Comment (bricesanchez) [inline config/locales/id.yml]: i think it's the wrong key lang. it should be `id:` instead `en:`

**Comment by bricesanchez** [comment]
Please fix your PR and check if CI works, it will be a pleasure to add your translation after :)

**Comment by rilut** [comment]
@bricesanchez Oops, sorry. Working on it.

**Comment by rilut** [comment]
Sorry, any idea on why the Postgre builds are green, while the MySQL builds are failing?

**Comment by bricesanchez** [comment]
@rilut No problems for MySQL, it's a known issue on mysql2 and rails : https://github.com/brianmario/mysql2/issues/675

**Comment by bricesanchez** [comment]
Could you squash your commits in one only ?

**Comment by rilut** [comment]
@bricesanchez ok, wait a sec

**Comment by rilut** [comment]
@bricesanchez done

**Comment by bricesanchez** [comment]
Thanks !

**Pull request merged**